### PR TITLE
Update Readme.md

### LINKED ISSUE #102: Create a readme of the project

### PR DESCRIPTION
**Pull request by SebastianBabinski1**
closed #102

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,136 +1,66 @@
...
+
+:woman_technologist: Weronika https://github.com/vieraboschkova
+
+:man_technologist: Dominik https://github.com/DominikNowak
+
+:woman_technologist: Aleksandra https://github.com/synowa
+
+:man_technologist: Sebastian https://github.com/Enessetere
+
+Special thanks for our mentor Aleksander!
```
Comment (vieraboschkova) [inline README.md]: change to: ..thanks to our...

**Review by vieraboschkova — CHANGES_REQUESTED** [review]
tiny spelling and good to go!

**Review by synowa — DISMISSED** [review]
Looks good

**Review by vieraboschkova — DISMISSED** [review]

**Review by synowa — APPROVED** [review]

**Review by Enessetere — APPROVED** [review]
well done

**Pull request merged**